src/raiden_libs/matrix.py

```diff
@@ -203,17 +205,17 @@ def _start_client(self) -> None:
         self.startup_finished.set()
 
     def follow_address_presence(self, address: Address, refresh: bool = False) -> None:
```

Comment:
Not new in this PR, but the `_displayname_cache`, `self._user_manager` and `follow_address_presence` could be pulled out into a separate class, which would make all the `if self._user_manager:` unnecessary.

Reply:
Let's make an issue for this.

Reply:
OK, I'll create one.

Reply:
@palango already did: https://github.com/raiden-network/raiden-services/issues/626